Google benchmark 1.0.0 needed

If you encounter the follow error:
```
[ 86%] Building CXX object osquery/CMakeFiles/osquery_benchmarks.dir/main/benchmarks.cpp.o
[ 86%] Linking CXX executable osquery_tables_tests
/Users/reed/git/github/osquery/osquery/main/benchmarks.cpp:17:34: error: cannot initialize a parameter of type 'const char **' with an lvalue of type 'char **'
  ::benchmark::Initialize(&argc, argv);
                                 ^~~~
/usr/local/include/benchmark/benchmark_api.h:155:41: note: passing argument to parameter 'argv' here
void Initialize(int* argc, const char** argv);
                                        ^
```

Please upgrade you google benchmark install.

### Linux
```bash
sudo rm /usr/local/lib/libbenchmark.a
sudo rm -rf /usr/local/include/benchmark
```

### OS X
```
$ brew remove --force google-benchmark
$ make deps
```

With version 1.0.0 on Darwin I see:
```
[ 97%] Linking CXX executable osquery_kernel_tests
Undefined symbols for architecture x86_64:
  "benchmark::internal::Benchmark::Benchmark(char const*, void (*)(benchmark::State&))", referenced from:
      __GLOBAL__sub_I_database_benchmarks.cpp in database_benchmarks.cpp.o
      __GLOBAL__sub_I_events_benchmarks.cpp in events_benchmarks.cpp.o
      __GLOBAL__sub_I_plist_benchmarks.cpp in plist_benchmarks.cpp.o
      __GLOBAL__sub_I_sql_benchmarks.cpp in sql_benchmarks.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [osquery/osquery_benchmarks] Error 1
make[2]: *** [osquery/CMakeFiles/osquery_benchmarks.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
[ 97%] Built target osquery_kernel_tests
[ 98%] Linking CXX executable osquery_kernel_benchmarks
Undefined symbols for architecture x86_64:
  "benchmark::internal::Benchmark::Benchmark(char const*, void (*)(benchmark::State&))", referenced from:
      __GLOBAL__sub_I_kernel_communication_benchmarks.cpp in kernel_communication_benchmarks.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [osquery/osquery_kernel_benchmarks] Error 1
make[2]: *** [osquery/CMakeFiles/osquery_kernel_benchmarks.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -142,9 +142,6 @@ else()
   endif()
 endif()
 
-# BUILD BUG: See issue #2112
-set(ENV{SKIP_BENCHMARKS} TRUE)
-
 # make debug (environment variable from Makefile)
 if(DEFINED ENV{DEBUG})
   set(DEBUG TRUE)

--- a/osquery/CMakeLists.txt
+++ b/osquery/CMakeLists.txt
@@ -339,7 +339,7 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   # osquery benchmarks.
   if(NOT DEFINED ENV{SKIP_BENCHMARKS} AND NOT ${OSQUERY_BUILD_SDK_ONLY})
     add_executable(osquery_benchmarks main/benchmarks.cpp ${OSQUERY_BENCHMARKS})
-    ADD_DEFAULT_LINKS(osquery_benchmarks FALSE)
+    ADD_DEFAULT_LINKS(osquery_benchmarks TRUE)
     target_link_libraries(osquery_benchmarks benchmark libosquery_testing)
     SET_OSQUERY_COMPILE(osquery_benchmarks "${GTEST_FLAGS} ${CXX_COMPILE_FLAGS}")
     set(BENCHMARK_TARGET "$<TARGET_FILE:osquery_benchmarks>")

